src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -512,6 +532,12 @@ public MetadataResolverRepository getEidasMetadataResolverRepository(
         return resolverRepository;
     }
 
+    @Provides
+    @Singleton
+    private SecretKeyEncrypter getSecretKeyEncrypter(MetadataBackedEncryptionCredentialResolver encryptionCredentialResolver) {
+        return new SecretKeyEncrypter(encryptionCredentialResolver);
+    }
```

Comment:
Provides the MSA private encryption key to `SecretKeyEncrypter`

Reply:
If these comments are helpful now, then can they be code comments please?

Reply:
AFAIK there is a no comment policy for Verify Hub, but I agree they may be useful in a few places.